docs/patterns/tools/use-tools/use-specific-actions.mdx

```diff
@@ -13,12 +13,14 @@ description: "Each Tool (like GitHub, Slack, etc.) comes with many Actions. You
 `GITHUB_CREATE_AN_ISSUE` and `GITHUB_COMMIT_EVENT` are action IDs for actions in the GitHub Tool
 <CodeGroup>
 ```python Python
-from composio import ComposioToolSet, App, action
+from composio_langchain import ComposioToolSet, Action
```

Comment:
The import statement has been changed from `composio` to `composio_langchain`, but this example appears to be in the general tools documentation rather than framework-specific docs. Consider keeping the base `composio` import here since this is showing general tool usage, and add a note about framework-specific imports in the respective framework docs.